release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -291,6 +291,34 @@ Currently, it is not supported to use Active Directory Federation Services with
 
 For the full list of OIDC providers, see xref:../authentication/identity_providers/configuring-oidc-identity-provider.adoc#identity-provider-oidc-supported_configuring-oidc-identity-provider[Supported OIDC providers].
 
+[id="ocp-4-11-auth-pod-security-admission"]
+==== Pod security admission autolabeling
+
+link:https://kubernetes.io/docs/concepts/security/pod-security-admission/[Pod security admission] is now enabled on {product-title}. 
+
+If the pod security admission plug-in is activated, {product-title} preserves pod workloads despite changes to the pod security API. A cluster performs this action by monitoring administrator-labeled namespaces for changes and mapping link:https://kubernetes.io/docs/concepts/security/pod-security-standards/[pod security standards] to xref:../authentication/managing-security-context-constraints.adoc[security context constraints].
+
+You can learn more about admission controller configuration in the link:https://kubernetes.io/docs/tasks/configure-pod-container/enforce-standards-admission-controller/#configure-the-admission-controller[Kubernetes documentation].
+
+The `privileged` pod security profile is enforced, and the `restricted` profile is used for warnings and audits.
+
+This release also introduces new security context constraints (SCCs) that enforce pod security in accordance with new pod security admission controls. These SCCs are:
+
+* `restricted-v2`
+* `hostnetwork-v2`
+* `nonroot-v2`
+
+They correspond to older, similarly named SCCs, but with the following enhancements:
+
+* `ALL` capabilities are dropped from containers.
+* The `NET_BIND_SERVICE` capability can be added explicitly.
+* `seccompProfile` is set to `runtime/default` by default.
+* `allowPrivilegeEscalation` must be unset or set to `false` in security contexts. 
+
+New clusters use the `restricted-v2` SCC in place of the `restricted` SCC that is available to any authenticated user. Clusters that are upgraded to 4.11 from previous versions allow both `restricted-v2` and `restricted` SCCs. 
```

Comment:
> New clusters use the `restricted-v2` SCC in place of the `restricted` SCC that is available to any authenticated user.

not being native, does this unambiguously mean that the `restricted` is no longer available to authenticated users in freshly installed clusters?

Reply:
```suggestion
New clusters use the `restricted-v2` SCC in place of the `restricted` SCC; the `restricted` SCC is no longer available to users of new clusters. Clusters that are upgraded to 4.11 from previous versions allow both `restricted-v2` and `restricted` SCCs. 
```